recipes/isa-l/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Use nasm as default assembler. Use relative path here as configure script
+# seems to choke on absolute paths using $CONDA_PREFIX.
+export AS=nasm
+export CFLAGS=""  # default CFLAGS are incompatible with nasm.
```

Comment:
This is probably wrong.  Can you figure out which flags are wrong and unset those?  The default flags are usually needed for proper ABI compatibility on conda-forge

Reply:
Yes, you are right. I was being lazy and frustrated. I have checked the build again and the CFLAGS are used for both gcc and nasm, while nasm does not have necessarily all flags compatible. It is of course important that gcc uses the correct cflags.

The isa-l package does not use nasm directly, but uses a nasm-filter script to filter incompatible options for nasm. I have included a patch that removes some of the flags that conda added, were incompatible, and slipped trough the original filter.